packages/orbit-components/src/Modal/README.md

```diff
@@ -35,11 +36,13 @@ Table below contains all types of the props available in the Modal component.
 | onClose             | `event => void \| Promise` |            | Function for handling onClose event. If you don't pass any function the Close button will not be displayed and it will not be possible to close the Modal. [See Functional specs](#functional-specs). |
 | preventOverlayClose | `boolean`                  |            | Property for preventing closing of modal when there is a action on overlay. BEWARE: This should be used only in very specials edge-cases! It breaks user experience.                                  |
 | hasCloseButton      | `boolean`                  | `true`     | Defines whether the Modal displays a close button. If you disable this, we recommend adding some kind of an alternative.                                                                              |
-| autoFocus           | `boolean`                  | `true`     | The autofocus attribute of the Modal, see [this docs](https://www.w3schools.com/tags/att_autofocus.asp).                                                                                              |
 | disableAnimation    | `boolean`                  | `false`    | Defines whether the Modal performs the slide in animation on mobile. If you want to improve your [CLS](https://web.dev/cls/) score, you might want to set this to `true`.                             |
 | mobileHeader        | `boolean`                  | `true`     | If `false` the ModalHeader will not have MobileHeader and CloseContainer.                                                                                                                             |
-| labelClose          | `string`                   | `Close`    | The label for the close button.                                                                                                                                                                       |
+| labelClose          | `string`                   |            | The label for the close button. It is required all the time, unless `hasCloseButton` is explicitly set to `false`.                                                                                    |
 | onScroll            | `event => void \| Promise` |            | Function for handling `onScroll` event. [See Functional specs](#functional-specs).                                                                                                                    |
+| ariaLabelledby      | `string`                   |            | The `aria-labelledby` attribute of the Modal. It should be used if `title` is not defined on the ModalHeader.                                                                                         |
```

Comment:
Is it preferred to use `aria-labelledby` over `aria-label`?

Reply:
From my understanding, yes. Because the Modal should have a visible title, so that we even have a `title` prop that handles this logic automatically. But the Modal can have a custom Header, that will not use the `title` prop but will still have a visible text title, hopefully. So the `ariaLabelledby` can refer to that "custom" title. And it is better to do so than to rely on `ariaLabel`.